waterlog/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=https://api.iot.retrotest.co.za 
```

Comment:
I am sure these go in the gitignore

Reply:
I mean these files

Reply:
Unless you are not going to use these env files to not store passwords? in which case seems like its fine and I will resolve

Reply:
These will never be used to store passwords. Only URL to specify the backend API

Reply:
This will never be used to store passwords.